github/repos.go

```diff
@@ -68,6 +68,7 @@ type Repository struct {
 	HasDownloads      *bool   `json:"has_downloads"`
 	LicenseTemplate   *string `json:"license_template,omitempty"`
 	GitignoreTemplate *string `json:"gitignore_template,omitempty"`
+	Archived          *bool   `json:"archived"`
```

Comment:
Why doesn't this field have `,omitempty` like the other fields above?

Reply:
Honestly, I wasn't sure about that and got confused by looking at the existing fields of type `*bool` some of which have `,omitempty` while others don't have. What's the logic/reasoning behind that?

Reply:
No worries.

> got confused by looking at the existing fields (in `Repository`) of type `*bool` some of which have `,omitempty` while others don't.

The reason is very relevant. It's primarily to have consistency, so other people don't run into such situation where it's unclear which of the two styles to use. As far as I can tell, it's much more common to use `,omitempty` on pointer fields than not. We should fix the other fields too (probably in a separate issue/PR).

There's very little effect of using/not using `,omitempty` here. Including it makes it so that fields with `nil` pointer values get omitted if the struct happens to get JSON marshaled.

Reply:
Thanks! Updated my commit. Please review.